### PR TITLE
Add Unified Mode deprecations page

### PR DESCRIPTION
**Pull request by lamont-granquist**
This is just the deprecations page which is focused on the user doing remediations.  We also need docs on the whole thing and better utilizing it.

### CODE REVIEW

File: content/deprecations_unified_mode.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+draft = false
+
+gh_repo = "chef-web-docs"
+robots = "noindex"
+
+
+aliases = "/deprecations_unified_mode.html"
++++
+
+Starting in Chef Infra 17.0 resources should be converted to use Unified Mode, which eliminates the split between
```
Comment (tas50) [inline content/deprecations_unified_mode.md]:
```suggestion
Starting in Chef Infra Client 17.0 resources should be converted to use Unified Mode, which eliminates the split between
```

File: content/deprecations_unified_mode.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+the compile and converge modes in the specific resource where it is declared.
+
+Unified Mode has been available since Chef Infra 14.14.14, Chef Infra 15.3.14 and Chef Infra 16.0 and all core chef resources
```
Comment (tas50) [inline content/deprecations_unified_mode.md]:
```suggestion
Unified Mode has been available starting with Chef Infra Client 14.14.14, Chef Infra Client 15.3.14, and Chef Infra 16.0 and all core chef resources
```

File: content/deprecations_unified_mode.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+in Chef Infra 17.0 have Unified Mode enabled.
```
Comment (tas50) [inline content/deprecations_unified_mode.md]: No one cares if we implemented it internally. Just nuke that

File: content/deprecations_unified_mode.md [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+
+Unified Mode is a setting which is local to the resource it is declared inside, and it does not affect
+calling recipes or resources, and does not affect any sub-resources which are used.
+
+The default behavior of Chef Infra 17 is that resources do not run in Unified Mode, which is the same behavior as
```
Comment (tas50) [inline content/deprecations_unified_mode.md]:
```suggestion
The default behavior of Chef Infra Client 17 is that resources do not run in Unified Mode, which is the same behavior as
```

File: content/deprecations_unified_mode.md [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+robots = "noindex"
+
+
+aliases = "/deprecations_unified_mode.html"
++++
+
+Starting in Chef Infra Client 17.0 resources should be converted to use Unified Mode, which eliminates the split between
+the compile and converge modes in the specific resource where it is declared.
+
+Unified Mode has been available starting with Chef Infra Client 14.14.14, Chef Infra Client 15.3.14, and Chef Infra 16.0 and all core chef resources
```
Comment (tas50) [inline content/deprecations_unified_mode.md]:
```suggestion
Unified Mode has been available starting with Chef Infra Client 14.14.14, Chef Infra Client 15.3.14, and Chef Infra Client 16.0.
```

File: content/deprecations_unified_mode.md [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+in Chef Infra 17.0 have Unified Mode enabled.
```
Comment (tas50) [inline content/deprecations_unified_mode.md]:
```suggestion
```

**Comment by lamont-granquist** [comment]
sous-chefs found a real-world case of where resources need to be updated.

**Comment by kagarmoe** [comment]
Closed by https://github.com/chef/chef-web-docs/pull/3111

**Pull request closed without merging**